client/api.lua

```diff
@@ -37,6 +37,24 @@ function api.addSphereZone(data)
     return lib.zones.sphere(data).id
 end
 
+---@param id number | string The ID of the zone to check. It can be either a number or a string representing the zone's index or name, respectively.
+---@return boolean returns true if the zone with the specified ID exists, otherwise false.
+function api.zoneExists(id)
+    if not Zones or (type(id) ~= 'number' and type(id) ~= 'string') then
+        return false
+    end
+
+    for key, zone in pairs(Zones) do
```

Comment:
When id is a number all you need is `Zones[id]`, rather than iterating over over the zones.